src/Angor/Client/Pages/Spend.razor

```diff
@@ -501,6 +513,11 @@
 
     private async Task ClaimCoins(int stageId)
     {
+        if (!targetInvestmentReached)
+        {
+            throw new InvalidOperationException("Target investment amount has not been reached, you can only release the funds back to the investors.");
```

Comment:
I am not sure about this, what if the target is close and the investors are happy for founder to continue?